Form-Controls/styles.css

```diff
@@ -0,0 +1,125 @@
+
+body{
+    background-color: rgb(213, 206, 210);
+    text-align: center;
+}
+header{
+  border-bottom: 1px solid #571f0c;
+}
+footer{
+  border-top: 1px solid #571f0c;
+}
+.form-section{
+    font-family: 'Roboto';
+    font-style: normal;
+    font-size: 20px;
+    width: 50%;
+   margin-top: 5rem;
+   margin-bottom: 5rem;
+     margin-left: 25%;
+    margin-right: 25%;
+}
+.fieldset-store{
+        border: none;
+        margin: 0;
+        padding: 0;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+        margin-left: 25%;
+        margin-right: 25%;
+}
+.form-input {
+    width: 100%;
+    height: 2rem;
+    margin-bottom: 1.5rem;
+    border: 2px solid #666666;
+border-radius: 4px;
+  }
+  .full-name{
+    width: 100%;
+  }
+.email{
+    width: 100%;
+}
+.date{
+    width: 100%;
+   
+}
+  select{
+    width: 100%;
+    height: 2.4rem;
```

Comment:
Unless you are super clear on what you are doing, steer clear of setting explicit heights on UI.  It can get complicated to handle later.